src/utility/json_import.rb

```diff
@@ -276,6 +282,17 @@ def add_pods(json_objects, nodes)
       end
     end
 
+    def add_mounted_users(json_objects, nodes, rotation_around_center)
+      return if json_objects['mounted_users'].nil?
+      json_objects['mounted_users'].each do |user_json|
+        node = nodes[user_json['id']]
+        node.hub.attach_user weight: user_json['weight'], filename: user_json['filename']
```

Comment:
This will fail on the server side because the weight is set as string (but should be int)
```suggestion
        node.hub.attach_user weight: user_json['weight'].to_i, filename: user_json['filename']
```